server/src/service/OfferService.cc

```diff
@@ -0,0 +1,135 @@
+#include "OfferService.hh"
+
+namespace server::service {
+
+    oatpp::Object<server::dto::OfferDto>
+    OfferService::createOne(
+            oatpp::Object<server::dto::OfferDto> const &dto
+    ) {
+        try {
+            this->getOne(dto->id); // Will throw 404 if not found
+            OATPP_ASSERT_HTTP(false, Status::CODE_409, "Offer already exists")
+        }
+        catch (oatpp::web::protocol::http::HttpError &e) {
+            if (e.getInfo().status != Status::CODE_404) { throw; }
```

Comment:
Again, why the if statement?

Reply:
see [this](https://github.com/mnurzyns/ParkSpaceSharing/pull/36/files/eaf878e5106238861b3c7c1f7a74ae917f92e110#r1179550193)